src/validator/client/README.md

```diff
@@ -0,0 +1,185 @@
+> This is a local copy of openapi-typescript-fetch
```

Comment:
skip this whole sub-module if you want. I stole this because it has some build issues, but the typescript code is perfectly good. it provides a really nice, typed fetch interface that works extremely well with our auto-build client from our openapi spec.